pkgs/applications/networking/instant-messengers/signal-desktop/signal-desktop-darwin.nix

```
@@ -0,0 +1,63 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  _7zz,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "signal-desktop";
+  version = "7.28.0";
+
+  src = fetchurl {
+    url =
+      with finalAttrs;
+      "https://updates.signal.org/desktop/signal-desktop-mac-universal-${version}.dmg";
+    hash = "sha256-yPObHZPCXRycuf9VexooWkXDJLReEiTuFSmCxlHA/OU=";
+  };
+  sourceRoot = ".";
+
+  nativeBuildInputs = [ _7zz ];
+
+  dontConfigure = true;
+  dontBuild = true;
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/Applications
+    cp -r Signal.app $out/Applications
+
+    runHook postInstall
+  '';
+
+  passthru = {
+    updateScript.command = [ ./update.sh ];
+  };
+
+  meta = {
+    description = "Private, simple, and secure messenger";
+    longDescription = ''
+      Signal Desktop is an Electron application that links with your
+      "Signal Android" or "Signal iOS" app.
+    '';
+    homepage = "https://signal.org/";
+    downloadPage = "https://signal.org/download/macos/";
+    changelog = with finalAttrs; "https://github.com/signalapp/Signal-Desktop/releases/tag/v${version}";
```

This is absolutely not needed and even longer... let's keep it simple:
```suggestion
    changelog = "https://github.com/signalapp/Signal-Desktop/releases/tag/v${finalAttrs.version}";
```